examples/private-input-module/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use hdp::{
+    hdp_run::{self, HdpRunConfig},
+    preprocessor::module_registry::ModuleRegistry,
+    primitives::task::{
+        module::{ModuleInput, Visibility},
+        TaskEnvelope,
+    },
+};
+
+#[tokio::main]
+async fn main() {
+    //set RUST_LOG into debug
+    std::env::set_var("RUST_LOG", "debug");
+
+    let module_regisry = ModuleRegistry::new();
+    let module = module_regisry
+        .get_extended_module_from_class_source(
+            None,
+            Some("./private_module/target/dev/private_module_get_balance.compiled_contract_class.json".into()),
+            vec![
+                ModuleInput::new(Visibility::Private, "0x5222a4"),
+                ModuleInput::new(Visibility::Public, "0x00000000000000000000000013cb6ae34a13a0977f4d7101ebc24b87bb23f0d5" )
+            ],
+        )
+        .await
+        .unwrap();
+    let tasks = vec![TaskEnvelope::Module(module)];
+    let pre_processor_output_file = "input.json";
+    let output_file = "output.json";
+    let cairo_pie_file = "pie.zip";
+
+    let hdp_run_config = HdpRunConfig {
+        dry_run_program_path: "../../build/contract_dry_run.json".into(),
+        sound_run_program_path: "../../build/hdp.json".into(),
+        pre_processor_output_file: pre_processor_output_file.into(),
+        processor_output_file: Some(output_file.into()),
+        cairo_pie_file: Some(cairo_pie_file.into()),
+        ..Default::default()
+    };
+
+    hdp_run::hdp_run(&hdp_run_config, tasks).await.unwrap();
```

Comment:
little bit weird syntax

Reply:
maybe smth like hdp::run() ?

Reply:
or hdp_run::run()

Reply:
i got idea from https://github.com/lambdaclass/cairo-vm/blob/93f1f54f38059af89850b1c06d4c901300f9b94d/vm/src/cairo_run.rs#L116 which i like than too general function name `run` i'll make export in upper module so user can call it as `hdp_run` not `hdp_run::hdp_run`

Reply:
`hdp::run`